store.go

```diff
@@ -2473,25 +2473,21 @@ func (s *store) DeleteImage(id string, commit bool) (layers []string, err error)
 		// Perform delete image on primary imageStore but if image
 		// is not found on primary store then try delete on any additional
 		// write-able image store.
```

Comment:
I think this comment is still inaccurate, https://github.com/containers/storage/pull/1578#discussion_r1174138580 .

Reply:
Fixed.